lib/src/chart/bar_chart/bar_chart_painter.dart

```diff
@@ -371,6 +376,23 @@ class BarChartPainter extends AxisChartPainter<BarChartData> {
     }
   }
 
+  void _paintShadow(
```

Comment:
Don't forget to write unit-test for this.

Reply:
@imaNNeoFighT  unfortunately I don't have any experience for this. Could you help me on it 🙂?

Reply:
Yes, for sure. I'm here to help.
I will give you some clues.

**Importance:**
First of all, you need to understand the unit-test importance. 
When we write logic (any logic, here we are painting something), we need to write a test function which verifies that logic is working correctly. 
All of these test functions will run automatically when anyone changes the code. In this way, we can realize if some changes on this logic break it or not automatically.

**Where to start:**
We have a [CONTRIBUTING](https://github.com/imaNNeoFighT/fl_chart/blob/master/CONTRIBUTING.md#test-coverage-unit-tests) doc which you can read about our unit tests system.

Then I suggest you read some of the unit tests under the `/tests` directory (start with simple functions)

Cheers!